content/en/docs/Endpoints/getTranscodeStream.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "getTranscodeStream"
+linkTitle: "getTranscodeStream"
+categories:
+- Transcoding
+description: >
+  Returns a transcoded media stream.
+---
+
+`http://your-server/rest/getTranscodeStream`
+
+Returns a transcoded media stream. **Note:** Clients should not construct this URL manually. Instead, they should use the `transcodeUrl` provided in the response of the [`getTranscodeDecision`](../gettranscodedecision) endpoint.
```

Comment:
Since we now have `transcodeParams`, the note is no longer valid (the client does have to construct the URL manually)